Does not compile with RN 0.47

In RN 0.47 there is no more createJSModules method in ReactPackage interface. Because of this KCKeepAwakePackage does not compile. To make it backwards compatible, just remove @override.

```
node_modules/react-native-keep-awake/android/src/main/java/com/corbt/keepawake/KCKeepAwakePackage.java:23: error: method does not override or implement a method from a supertype
    @Override
```

[ReactPackage.java](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/ReactPackage.java)

--- a/android/src/main/java/com/corbt/keepawake/KCKeepAwakePackage.java
+++ b/android/src/main/java/com/corbt/keepawake/KCKeepAwakePackage.java
@@ -20,7 +20,7 @@ public class KCKeepAwakePackage implements ReactPackage {
         return modules;
     }
 
-    @Override
+    // Deprecated RN 0.47
     public List<Class<? extends JavaScriptModule>> createJSModules() {
         return Collections.emptyList();
     }

